### DISCUSSION TITLE
[公式快] 部分公式无法显示(3.16.4)

### DISCUSSION THREAD

**Discussion by 3364fy** [tangly1024/NotionNext#1972] [Q&A]
**描述bug**
![image](https://github.com/tangly1024/NotionNext/assets/99535675/91bd42e7-cef5-4a2a-b3e8-6b9385b3faad)
notion中的公式块在网页中不显示
![image](https://github.com/tangly1024/NotionNext/assets/99535675/9cedf61a-403b-477c-8307-66f8d016a9b5)

**环境**

- 操作系统:  windows
- 浏览器 edge

**Comment by tangly1024** [comment]
请提供对应的网页链接

**Comment by 3364fy** [comment]
> 

https://www.3364fy.top/article/stress

**Comment by tanglyqq** [comment]
可能是3.16.4 版本中的一个bug，在最新版中，我的公式显示是正常的。


建议更新到最新版本，因为3.16.4版本不再维护。

预览效果
```
\tau_{xy}=-\tau_{yx}\tau_{xz}=-\tau_{zx}\tau_{yz}=-\tau{zy}
```
https://preview.tangly1024.com/article/example-1#7de9f2bb6e8b4a0db12662148de95bd5

![image](https://github.com/tangly1024/NotionNext/assets/107399839/40cd5ffb-815f-45ce-a952-90f3da990627)

![image](https://github.com/tangly1024/NotionNext/assets/107399839/9becd3ce-6826-4a92-b578-ade643bd7547)
